executors.go

```diff
@@ -36,6 +42,10 @@ func (q *Query) ExecWithCount() (int, error) {
 	count := int64(0)
 	return int(count), q.Connection.timeFunc("Exec", func() error {
 		sql, args := q.ToSQL(nil)
+		if sql == "" {
+			return fmt.Errorf("empty query")
+		}
```

Comment:
if the `sql`, the result of `ToSQL()` is empty, just abort the execution. This is the main part of this PR.